runtime/dancebox/src/migrations.rs

```diff
@@ -209,19 +210,63 @@ where
     }
 }
 
+pub struct MigrateConfigurationFullRotationPeriod<T>(pub PhantomData<T>);
+impl<T> Migration for MigrateConfigurationFullRotationPeriod<T>
+where
+    T: pallet_configuration::Config,
+{
+    fn friendly_name(&self) -> &str {
+        "TM_MigrateConfigurationFullRotationPeriod"
+    }
+
+    fn migrate(&self, _available_weight: Weight) -> Weight {
+        log::info!(target: LOG_TARGET, "migrate");
+
+        Configuration::set_full_rotation_period(RuntimeOrigin::root(), 24u32)
+            .expect("Failed to set full_rotation_period");
+        <T as pallet_configuration::Config>::WeightInfo::set_config_with_u32()
+    }
+
+    /// Run a standard pre-runtime test. This works the same way as in a normal runtime upgrade.
+    #[cfg(feature = "try-runtime")]
+    fn pre_upgrade(&self) -> Result<Vec<u8>, sp_runtime::DispatchError> {
+        let zero_period = Configuration::config().full_rotation_period;
```

Comment:
this should indeed panic with try-runtime no? the config does not still have `full_rotation_period`

Reply:
I am assuming it defaults to 0

Reply:
In this case what happens is that `Configuration::config()` returns the default value, I am assuming because it is a ValueQuery. The default value has full_rotation_period = 24, so this test was wrong.